apps/blockchain-mailer/src/service/mailService.ts

```diff
@@ -0,0 +1,43 @@
+import MailgunTransport from '../util/MailgunTransport';
+import nodemailer, { Transporter } from 'nodemailer';
```

Comment:
External package declarations go above local imports (and builtins should be first).

Reply:
Got it!